starboard/shared/starboard/player/BUILD.gn

```diff
@@ -71,3 +71,17 @@ action("player_download_test_data") {
     rebase_path("$outdir/testdata", root_build_dir),
   ]
 }
+
+copy("local_player_test_files") {
+  install_content = true
+  subdir = "starboard/shared/starboard/player"
+  sources = [
+    "testdata/flac.dmp",
+    "testdata/hevc.dmp",
+    "testdata/mp3.dmp",
+    "testdata/pcm.dmp",
+    "testdata/vorbis.dmp",
+    "testdata/vp8.dmp",
+  ]
```

Comment:
If this CL is checked in, we probably want to move these files to GCS and modify //starboard/shared/starboard/player/testdata/sha1_files.gni to include them. The added dmp files are ~1.6MB total.